API and Postman 💖 (#4434)
DataDog/documentation: content/en/api/overview/overview.md

external_redirect: /api/#overview
---
## Overview

> This section details Datadog's HTTP API. It makes it easy to get data in and out of Datadog.

Reviewer: ```suggestion
This section details Datadog's HTTP API for getting data in and out of Datadog.
```

> **In order to get started on Datadog HTTP API, use our [Datadog Postman collection][1]**

Reviewer: ```suggestion
**To get started on Datadog HTTP API, use our [Datadog Postman collection][1]**
```
